### DISCUSSION TITLE
[Release Vote] Vote for Release V0.5.0-rc1

### DISCUSSION THREAD

**Discussion by Aireed** [apache/amoro#1674] [Polls]
Hi Everyone,

I propose that we release the following RC as the official Arctic 0.5.0 release.

The commit ID is 5e1c7f5b23cf7fc923d2ae4cf09039c75421fda3

- This corresponds to the tag: v0.5.0-rc1
- https://github.com/NetEase/arctic/commits/v0.5.0-rc1
- https://github.com/NetEase/arctic/tree/v0.5.0-rc1

Please check out, build, and test.

[ ] +1 Release this as Arctic 0.5.0
[ ] +0
[ ] -1 Do not release this because...

Please vote in the next 72 hours and add your test steps when voting.

**Comment by wangtaohz** [comment]
+1.

I have deployed AMS locally and followed the steps in the "Get Started" guide, and tested the following features:

- mysql init sql
- create the optimizer group (local container) and start a local optimizer
- create internal catalog(both Mixed Iceberg and Iceberg)
- spark insert overwrite/flink cdc into Mixed Iceberg Table
- table optimizing and result
- verify the `self-optimizing.full.trigger.interval` for Iceberg Format Table

but there are still some bug/improvement should be fixed:
 - https://github.com/NetEase/arctic/issues/1691
 - https://github.com/NetEase/arctic/issues/1690

**Comment by HuangFru** [comment]
-1.
I have reproduced an error in the optimizer:
#1696 
This will influence the Iceberg format's self-optimizing.

**Comment by Aireed** [comment]
-1.

I have deployed AMS in server   and tested the following features:

- mysql init sql
- create the optimizer group (flink container) and start a flink optimizer
- create internal catalog(mixed hive & mixed iceberg)
- spark insert overwrite/flink cdc into mixed hive/Mixed Iceberg Table
- table optimizing and result

but there are still some bug should be fixed:

- #1685 
- #1698

**Comment by Aireed** [comment]
For #1696  is critical bug, so i close the vote and  will track the progress of bug fixes and release a vote for RC2 after testing validation.
